frontend/src/features/projects/spl/steps/GreTransferStep.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
```

Comment:
Same comment here.

Reply:
Component has been placed in common shared components